demo/src/screens/SettingsScreen.js

```diff
@@ -15,9 +14,11 @@ class SettingsScreen extends Component {
 
     const data = props.navigationData || navigationData;
     const playground = props.playground || playgroundScreen;
-    
+    const extraSettingsUI = props.extraSettingsUI;
```

Comment:
This part is also redundant, no need to store this prop in state cause we're rendering it directly from props.